CHANGES.md

```diff
@@ -3,6 +3,9 @@ Change Log
 
 ### 1.51 - 2018-11-01
 
+##### Breaking Changes :mega:
+* Renamed Atmosphere Sandcastle example to [Ground Atmosphere](https://cesiumjs.org/Cesium/Apps/Sandcastle/index.html?src=Ground%20Atmosphere.html) and Atmosphere color Sandcastle example to [Sky Atmosphere](https://cesiumjs.org/Cesium/Apps/Sandcastle/index.html?src=Sky%20Atmosphere.html).
```

Comment:
Sandcastle examples do not qualify as breaking changes, they are completely outside of the API.

Reply:
Hmm that's true... @OmarShehata could you remove the similar breaking change note from `1.50`?